wallet/src/main/scala/org/bitcoins/wallet/api/FeeProvider.scala

```diff
@@ -0,0 +1,16 @@
+package org.bitcoins.wallet.api
+
+import org.bitcoins.core.wallet.fee.FeeUnit
+
+import scala.concurrent.Future
+
+trait FeeProvider {
```

Comment:
As i have said in slack, I think we should require external fees be provided via methods for apis that require fees for now. I think we should delete this